### DISCUSSION TITLE
Server side issue

### DISCUSSION THREAD

**Discussion by DiegoDerganz** [Ysurac/openmptcprouter#3523] [Q&A] [answered]
I have some service on my VPS that I want to preserve.
After I have installed OMR Server stop working SSH on port 22 and Apache on port 80.
It's possible to restore it?

**Comment by Ysurac** [comment, marked as answer]
SSH is on port 65222, you can change it again in ```/etc/ssh/sshd_config```
You need to open ports in ```/etc/shorewall/rules``` and restart shorewall service.
